internal/ramses/ohbse.go

```diff
@@ -0,0 +1,59 @@
+package ramses

```

Comment:
To me the filename is strange, OHB (the company) SE (the country), seems too generic for keeping the internal structure of one of the headers. "tmheader.go" would be good?

Reply:
I know but as I understood it the header is the OHB (from Sweden)'s implementation of CCSDS standards for a TM header within the ramses specification. Kind of like when we wrote an extension to the RTZ in another project. "OHBSE CCSDS TM Packet" (header) is the actual name of it. My reservation for TM header is that it sounds like something that be part of / inside the telemetry packet, but as far as I undrestand this since it's part of the ramses spec, it's actually outside it.

Reply:
Even more problematic is that in the ramses specification there are various "TM Packet" (it doesn't use the word header) so I think it would be problematic if we don't in some way explicitly say that its an OHBSE CCSDS TM Packet and a compromise would be to keep the file name as `ohbse.go` and call it `TMPacket` (omit the header entirely). CCSDS reference might not be important enough to be in either but just in documentation.

Reply:
or we call the file `ohbse_ccsds_tm_packet.go` since files are not used in the code